runtime/src/main/java/io/quarkiverse/logging/splunk/SplunkConfig.java

```diff
@@ -165,6 +165,12 @@ public class SplunkConfig {
     @ConfigItem
     public Map<String, String> metadataFields = new HashMap<>();
 
+    /**
+     * Enables the "raw" mode
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean raw;
```

Comment:
Could you also edit the Javadoc of url `@ConfigItem` to mention the path for raw endpoint ?

Reply:
Updated, thanks

Reply:
If I am not wrong, we copy the generated content of the maven plugin in this file, check the header of config.adoc.

Reply:
Done, thanks